test/unit_tests/graphql/test_processes.py

```diff
@@ -81,6 +81,70 @@ def get_processes_query(
     ).encode("utf-8")
 
 
+def get_processes_query_with_subscriptions(
+    first: int = 10,
+    after: int = 0,
+    filter_by: Union[list[str], None] = None,
+    sort_by: Union[list[dict[str, str]], None] = None,
+) -> bytes:
+    query = """
+query ProcessQuery($first: Int!, $after: Int!, $sortBy: [GraphqlSort!], $filterBy: [GraphqlFilter!]) {
+  processes(first: $first, after: $after, sortBy: $sortBy, filterBy: $filterBy) {
+    page {
+      assignee
+      createdBy
+      failedReason
+      isTask
+      lastStep
+      traceback
+      customer
```

Comment:
I might be wrong about this: when customer will be removed from subscription in the GPL version. This should also be removed?